bin/install

```diff
@@ -67,7 +81,7 @@ get_download_url() {
   local filename
   filename="$(get_filename "$version" "$platform" "$binary_name")"
 
-  echo "https://github.com/robscott/kube-capacity/releases/download/${version}/${filename}"
+  echo "https://github.com/robscott/kube-capacity/releases/download/v${version}/${filename}"
```

Comment:
This needs to be conditional upon the version attempting to be downloaded. Prior to `v0.5.0` the `v` should not be included as part of the version. Again refer to https://gitlab.com/wt0f/asdf-fx/-/blob/main/bin/install?ref_type=heads#L63-72 to see how I handle this sort of issue.